cpu/sam3/periph/dac.c

```diff
@@ -64,8 +64,8 @@ void dac_poweron(dac_t line)
 {
     assert(line < DAC_NUMOF);
 
-    PMC->PMC_PCER1 = (1 << (ID_DACC - 32));
-    DACC->DACC_CHER = PMC_BIT;
+    DACC->DACC_CHER = (1 << line);
+    PMC->PMC_PCER1 = PMC_BIT;
```

Comment:
Mhhhh this changes the functionality. From the surrounding lines I suspect this is correct, but I'm not sure. @haukepetersen can you confirm, please?

Reply:
It would make the review easier if this was a separate PR.

Reply:
:+1:

Reply:
okay I'll factor out.